client/GameComponents/Card.jsx

```diff
@@ -132,6 +133,7 @@ class Card extends React.Component {
         var counters = {};
 
         counters['card-fate'] = card.fate ? { count: card.fate, fade: card.type === 'attachment', shortName: 'F' } : undefined;
+        counters['card-honor'] = card.honor ? { count: card.honor, fade: card.type === 'attachment', shortName: 'F' } : undefined;
```

Comment:
I don't think card.honor is something which is used by either the client or the server, but I'm assuming this is here for future-proofing!  I think if we ever needed to put honor on anything else, card.tokens is probably the best way to do it.